test/ui-pytest/scripts/utils/local_system.py

```diff
@@ -1,124 +1,17 @@
-        try:
-            for conns in proc.connections(kind='inet'):
-                if conns.laddr.port == port:
-                    return process_info(
-                        proc.pid,
-                        proc.name(),
-                        datetime.fromtimestamp(proc.create_time()).strftime("%H:%M:%S.%f")
-                    )
-        except (psutil.NoSuchProcess, psutil.AccessDenied, psutil.ZombieProcess):
-            pass
-
-
-@allure.step('Kill process by name')
-def kill_process_by_name(process_name: str, verify: bool = True, timeout_sec: int = 10):
-    _logger.info(f'Closing process: {process_name}')
-    processes = find_process_by_name(process_name)
-    for process in processes:
-        try:
-            os.kill(process.pid, signal.SIGILL if IS_WIN else signal.SIGKILL)
-        except PermissionError as err:
-            _logger.info(f'Close "{process}" error: {err}')
-    if verify and processes:
-        wait_for_close(process_name, timeout_sec)
-
-
-@allure.step('Kill process by PID')
-def kill_process_by_pid(pid, verify: bool = True, timeout_sec: int = 10):
-    os.kill(pid, signal.SIGILL if IS_WIN else signal.SIGKILL)
-    if verify:
-        wait_for_close(pid=pid, timeout_sec=timeout_sec)
-
-
-@allure.step('Kill process by port')
-def kill_process_by_port(port: int):
-    proc = find_process_by_port(port)
-    if proc is not None and proc.pid:
-        kill_process_by_pid(proc.pid)
-
-
-@allure.step('Wait for process start')
-def wait_for_started(process_name: str = None, timeout_sec: int = configs.timeouts.PROCESS_TIMEOUT_SEC):
-    started_at = time.monotonic()
-    while True:
-        process = find_process_by_name(process_name)
-        if process:
-            _logger.info(f'Process started: {process_name}, start time: {process[0].create_time}')
-            return process[0]
-        time.sleep(1)
-        _logger.debug(f'Waiting time: {int(time.monotonic() - started_at)} seconds')
-        assert time.monotonic() - started_at < timeout_sec, f'Start process error: {process_name}'
-
-
-@allure.step('Wait for process close')
-def wait_for_close(process_name: str = None, timeout_sec: int = configs.timeouts.PROCESS_TIMEOUT_SEC, pid=None):
-    started_at = time.monotonic()
-    while True:
-        if process_name is not None:
-            process = find_process_by_name(process_name)
-            if not process:
-                break
-        elif pid is not None:
-            process = find_process_by_pid(pid)
-            if process is None:
-                break
-        else:
-            raise RuntimeError('Set process name or PID to find process')
-        time.sleep(1)
-        assert time.monotonic() - started_at < timeout_sec, f'Close process error: {process_name or pid}'
-    _logger.info(f'Process closed: {process_name}')
+@allure.step('Kill process')
+def kill_process(pid, verify: bool = True, timeout_sec: int = 10):
+    run(['kill', '-9', pid], check=verify, timeout_sec=timeout_sec)

```

Comment:
Same here. Why are we calling `kill` instead of using `os.kill()` method?

Reply:
I tested a few times, and the current solution of the closing  the process is more stable

Reply:
Weird but okay.